task-2/public/styles/scss/variables/mixins.scss

```diff
@@ -0,0 +1,61 @@
+@mixin header-logo {
```

Comment:
Most of your mixins are adding some text styling but its names are confusing because you can't see its purpose from its names.